RegisterController.js

```diff
@@ -37,9 +37,10 @@ export class RegisterController {
     if(!this._debounceExists) {
       this._debounceExists = pDebounce(async () => {
         let exists = false;
+        const email = this.state.email.toLowerCase();
```

Comment:
Another approach here is to modify the state itself so that the email changes on the UI as well. I'm not sure if that's better though -- think this approach is probably what we want and less jarring for a user. I'm just mentioning it in case it wasn't considered.

Reply:
hmm I actually didn't consider that and it's a good approach I think. Since we don't want to surprise the users and they can see exactly what they are posting. Should I make changes so the form field will only accept lower case chars, if that's okay?

Reply:
So that's yet another way to do it -- I think we should get some additional opinions on this to see what people think the best UX would be. We could check with some other popular sites as well to see what they do.

Reply:
I looked at some popular sites (such as FB, IG, Yahoo, Google etc) and seems like they do not change the email to lowercase on their UI when registering for an account. They accept caps on the UI and I think normalizing email is being handled on their server, I guess that's the best UX. So, I think what we have is fine and no changes are required.